Makefile.inc

```diff
@@ -254,6 +254,9 @@ endif
 LDFLAGS +=	-Wl,-rpath,$(LIBDIR)
 endif
 # LDFLAGS for all 
+ifeq ($(OS),win32)
+  LDFLAGS +=	-static-libgcc -static-libstdc++
```

Comment:
Please explain why this is needed.
My windows build run just fine on MINGW (on Linux)